packages/grafana-data/src/themes/colors.ts

```diff
@@ -0,0 +1,49 @@
+export const colors = {
```

Comment:
Wonder why is all this UI stuff now in grafana-data?

Reply:
@Clarity-89 because the types where already here and keeping the types & implementation (real values) in two different packages whas really painful, and we use it in data and have had to mock the whole theme

Reply:
Would it make sense to move the type definitions to grafana-ui then? Seems a bit weird to have all the UI values in the data package, but maybe it's just me.